src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -32,8 +34,7 @@ function WorkspaceWorkflowsPage({policy, route}: WorkspaceWorkflowsPageProps) {
     const {isSmallScreenWidth} = useWindowDimensions();
     const {isOffline} = useNetwork();
 
-    const ownerPersonalDetails = ReportUtils.getDisplayNamesWithTooltips(OptionsListUtils.getPersonalDetailsForAccountIDs([policy?.ownerAccountID ?? 0], CONST.EMPTY_OBJECT), false);
-    const policyOwnerDisplayName = ownerPersonalDetails[0]?.displayName;
+    const policyApprover = policy?.approver;
```

Comment:
I think we still wanna keep the default as `ownerPersonalDetails` if there's no explicit approver set in the policy

Reply:
The default approver is still policy's owner.
You can test this by disabling approvals, and then re-enable them.